<?xml version="1.0"?>
<launch>
  <node pkg="nodelet" type="nodelet" name="collision_avoidance_manager" args="manager" output="screen" />


  <!-- Pass in collision_avoidance_config_file:=/path/to/config.yaml to change options. -->
  <arg name="collision_avoidance_config_file"
     default="$(find collision_avoidance)/config/collision_avoidance_uav.yaml" />

  <node pkg="nodelet" type="nodelet" name="collision_avoidance_nodelet" args="load collision_avoidance/CollisionAvoidanceNodelet collision_avoidance_manager" output="screen">
      <rosparam file="$(arg collision_avoidance_config_file)" command="load" />

      <remap from="map" to="" />
      <remap from="sensor" to="" />
      <remap from="odometry" to="" />
      <remap from="imu" to="" />
      <remap from="control" to="" />
      <remap from="move_to" to="" />

      <param name="robot_frame_id" value="base_link" />
  </node>

  <!-- <node pkg="nodelet" type="nodelet" name="laserscan_to_pointcloud_nodelet" args="load laserscan_to_pointcloud/LTP collision_avoidance_manager" output="screen">
    <param name="scan_topic" type="str" value="scan" />
    <param name="cloud_topic" type="str" value="cloud" />
  </node> -->

</launch>
